grammar/argextend.mlp

```diff
@@ -11,8 +11,14 @@ open Q_util
 let loc = Ploc.dummy
 let default_loc = <:expr< Loc.ghost >>
 
+IFDEF STRICT THEN
+  let ploc_vala x = Ploc.VaVal x
+ELSE
+  let ploc_vala x = x
+END
+
```

Comment:
I'd rather put this definition in `grammar/q_util.mlp` than in this file that happens by accident to be opened by the other grammar modules.